chapters/emergencystop.adoc

```diff
@@ -0,0 +1,29 @@
+== Emergency stop
+
+This rule only applies to Division A.
+
+.Definition
+
+A team can ask to stop the game immediately after a grace period of 10 seconds or at the next stoppage, whichever happens first regardless of the current situation. It will receive a yellow card for this and must take a timeout immediately. If the team is out of timeouts, it is still allowed to remove robots from the field, but can not use any remaining timeout time.
+
+When the game is stopped due to this rule, there are three possibilities that may have happened:
+
+. 10 seconds have passed and the game is stopped;
+. The game is stopped earlier than 10 seconds due to the ball leaving the field or a foul occurred;
+. The human referee stopped the game manually earlier than 10 seconds.
+
+For these possibilities there are two methods to proceed the game:
+
+. For *1* and *3*, the game is continued with a free kick for the opposing team;
+. For *2*, the game is continued like a regular timeout.
+
+.Usage
+
+An emergency stop intent can be made by:
+
+// To inform the referee a communication flag shall be used
+// TODO: link "informing" with the communication flag section
+
+. A <<Robot Handler, robot handler>> by informing the <<Referee, referee>> who shall stop the game within 10 seconds or at the next stoppage.
```

Comment:
stop the game at the next stoppage? :D If the emergency stop is requested, the game should not be stopped later than the 10s

Reply:
Oops :smile: